avocado/utils/network/interfaces.py

```diff
@@ -176,6 +176,19 @@ def is_link_up(self):
         """
         return self.is_admin_link_up() and self.is_operational_link_up()
 
+    def get_default_interface(self):
```

Comment:
to avoid ambiguity, I suggest calling this  `get_default_route_iface` or `get_default_route_interface()`.